zcash_primitives/src/transaction/mod.rs

```diff
@@ -17,20 +17,19 @@ use std::convert::TryFrom;
 use std::fmt::Debug;
 use std::io::{self, Read, Write};
 use std::ops::Deref;
-use zcash_encoding::{CompactSize, Vector};
 
-use crate::{
+use ::sapling::{self, builder as sapling_builder};
```

Comment:
```suggestion
use ::sapling::builder as sapling_builder;
```
Or we could just use `sapling::builder` everywhere now, like we do `orchard::builder`.